tests/toranj/openthread-core-toranj-config-simulation.h

```diff
@@ -45,7 +45,7 @@
 
 #endif
 
-#define OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE 0
+#define OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE 1
```

Comment:
`OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE` is needed for CSL feature and the tests added in `test-604-channel-manager-csl.py`. Apparently, this causes other issues. What is the recommendation for csl related continous integration tests?

Reply:
I think they are all run by `simulation-1.1.yml` or `simulation-1.2.yml` workflows (in `.github/workflows`). Some may also run from other other workflows (`posix.yml`, `otns` etc.)